app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -131,17 +140,25 @@ Examples of eligible languages commonly taught in English state secondary school
 * Russian
 * Spanish
 
-State schools are schools funded by the UK government. Secondary schools teach children aged 11 to 16 or 11 to 18. You will not be eligible if you teach in a school for pupils aged 16 to 18 only.
+Teaching English is not classed as an eligible subject for the IRP
+
+You will not be eligible if you are:
+
+* employed by a recruitment agency – an agency can find you a job in a school, but your contract must be with the school and your salary must be paid directly to you by the school, not by the recruitment agency
+
+* teaching in Scotland, Wales or Northern Ireland
```

Comment:
```suggestion
* teaching in Wales, Scotland or Northern Ireland
```
suggest re-ordering - we normally use this order and it is consistent with the bullet below